UltiSnips/cpp.snippets

```diff
@@ -196,5 +196,33 @@ BOOST_AUTO_TEST_CASE(${4:TestCaseName})
 
 BOOST_AUTO_TEST_SUITE_END()
 
+endsnippet
+
+snippet transform "ranges::views::transform"
+${1:${2:std::}${3:ranges::}views::}transform($4)
+endsnippet
+
+snippet filter "ranges::views::filter"
+${1:${2:std::}${3:ranges::}views::}filter($4)
+endsnippet
+
+snippet r "ranges::"
+ranges::
+endsnippet
+
+snippet rv "ranges::views::"
+ranges::views::
+endsnippet
+
+snippet ra "ranges::actions::"
+ranges::actions::
+endsnippet
+
+snippet sr "std::ranges::"
+std::ranges::
+endsnippet
+
+snippet sv "std::views::"
+std::views::
```

Comment:
These don't use any ultisnips features, could you add them using the snipmate format instead? Thanks

Reply:
You mean moving them [in the other directory](https://github.com/honza/vim-snippets/tree/master/snippets) and using the format I find there?

Reply:
Yup!